src/renderer/components/Transaction/TransactionModal.vue

```diff
@@ -142,8 +144,8 @@ export default {
         if (this.isSuccessfulResponse(response)) {
           this.storeTransaction(this.transaction)
 
-          if (data && data.accept.length === 0 && data.broadcast.length > 0) {
-            this.$warn(messages.warningBroadcast)
+          if (data && (data.accept.length === 0 || data.broadcast.length === 0)) {
+            this.$warn(messages.warningLowFee, 10000)
```

Comment:
What you mean "hardcoding"?

Reply:
"10000"

Reply:
Oh, `10000` is not a fee but timeout for the warn message. I thought the default `5s` is not enough for the user to read the whole message, since it's kind of important... Not sure if this is the correct way.

Reply:
Yeh that makes sense. Maybe I'm wrong about it being that value, I think that's what Juan was referring to though 🤔